modules/motions/AngleRotation.h

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file   AngleRotation.h
+ * @brief  角度指定回頭動作
+ * @author Hara1274
+ */
+
+#ifndef ANGLE_ROTATION_H
+#define ANGLE_ROTATION_H
+
+#include "Rotation.h"
+#include "Mileage.h"
+#include "SystemInfo.h"
+
+class AngleRotation : public Rotation {
+ public:
+  /**
+   * コンストラクタ
+   * @param _robot       ロボット制御クラスへの参照
+   * @param _speed       指定する速度（mm/秒）
+   * @param _isClockwise 回頭方向 true:時計回り, false:反時計回り
+   * @param _targetAngle 目標回転角度(deg) 0~360
+   */
+  AngleRotation(Robot& _robot, double _speed, bool _isClockwise, int _targetAngle);
```

Comment:
引数の順番は、目標回転角度、速度、回頭方向にしてほしいかな
他の動作も基本的にこの順番に近い形になってて、これから調整の時に紛らわしくないようによろしくー

```suggestion
  /**
   * コンストラクタ
   * @param _robot       ロボット制御クラスへの参照
   * @param _targetAngle 目標回転角度(deg) 0~360
   * @param _speed       指定する速度（mm/秒）
   * @param _isClockwise 回頭方向 true:時計回り, false:反時計回り
   */
  AngleRotation(Robot& _robot, int _targetAngle, double _speed, bool _isClockwise);
```